db/versions/0017-migration.sql

```diff
@@ -0,0 +1,34 @@
+begin
+  -- lock this table before reading from it, to prevent loss of concurrent
+  -- updates when the table is dropped.  Note that this may lead to concurrent
+  -- updates failing; the important thing is that they not succeed without
+  -- taking effect.  Failed updates will be retried.
+  lock table wmworker_pool_errors_entities;
+
+  create table worker_pool_errors
+  as
+    select
+      (value ->> 'errorId')::text as error_id,
```

Comment:
`errorId` is defined as a SlugId in data.js. I think Dustin recommended moving away from azure's way of storing things as uuids but rather store things as slug ids. We'll want to modify the migration and downgrade script to make sure we have the proper slugid -> uuid and uuid -> slugid mappings. Dustin created two db functions that we can use `slugid_to_uuid` and `uuid_to_slugid`. We'll want to use them in both the migration and downgrade script. You can refer to Dustin's PR for the inspiration: https://github.com/taskcluster/taskcluster/pull/3115.